pkg/endpoint/endpoint.go

```diff
@@ -2368,18 +2368,15 @@ func (e *Endpoint) identityResolutionIsObsolete(myChangeRev int) bool {
 }
 
 // Must be called with e.Mutex NOT held.
-func (e *Endpoint) runLabelsResolver(owner Owner, myChangeRev int) {
-	// NOTE: UnconditionalLock is used here only for logging
-	e.UnconditionalLock()
```

Comment:
@ianvernon This change is not in any of the upstream PRs

Reply:
Fixed.